adagrams/game.py

```diff
@@ -1,11 +1,203 @@
+    if len(word) > 6:                                           # 8 
+        total_points_scored += 8                                # 9
+        
+    return total_points_scored                                  # 10
+    
+    
+
+# 1. function takes in a word as a parameter
+#    word, which is a string of characters
+
+# 2. score_chart
+#    using a dictionary - the key is immutable
+#    you can loop through both the keys and the values
+#    and return the key, which is an integer in this case.
+
+
+# 3   Each letter within word has a point value, that we'll need to sum later,
+#     so the variable total_points_scored helps us hold a value
+    
+# 4   Loop through each of the letters in the word that's provided as an argument
+#     for each word, if the user enters a word that's lowercase,
+#     using .upper() will capitalize it so it matches the case of the
+#     elements that are contained in the score_chart list
+
+# 5   loop through the dictionary's values-- which are a list
+#     Loop through both keys and values, by using the items() method:
+#     for x, y in thisdict.items():
+#        print(x, y)
+    
+# 6    check if that letter is in the dictionary's value     
+    
+# 7    and if it is, then add the dictionary's key- which is an integer
+#      to the total points scored
+
+# 8    if the length of the word is 7, 8, 9, or 10
+
+# 9    then add 8 additional points
+    
+# 10   the function returns an integer, representing the number of the total points scored
+
+
+# =============================== get_highest_word_score ===============================
+
+def get_highest_word_score(word_list):                  # 1
+    highest_score =  0                                  # 2
+    winning_word_list= []                               # 3
+	
+    for word in word_list:                              # 4
+
+        word_score = score_word(word)                   # 5
+
+        if word_score > highest_score:                  # 6
+            highest_score = word_score                  # 7
+            winning_word_list= [word]                   # 8
+        
+        
+        elif word_score == highest_score:               # 9
+            winning_word_list.append(word)              # 10
 
-def score_word(word):
-    pass
+    letter_count = 10                                   # 11
+    best_word = ''                                      # 12
+    
+    if len(winning_word_list) == 1:                     # 13
+        return winning_word_list[0], highest_score
+    else:                                               # 14
+        for word in winning_word_list:
 
-def get_highest_word_score(word_list):
-    pass
\ No newline at end of file
+            if len(word) == 10:                         # 15
+                return word, highest_score
+            else:
+                
+                if len(word) < letter_count:            # 16
+                    letter_count = len(word)
+                    best_word = word
+    
```

Comment:
Okayyyy Stacey! You ateee! You did a great job with this project! If you have any questions or comments aboutthe feedback I left please feel free to reach out! Once again great job! ⭐️